v1/ao/trace.go

```diff
@@ -147,19 +154,27 @@ func NewTraceWithOptions(spanName string, opts SpanOptions) Trace {
 // incoming trace ID (e.g. from a incoming RPC or service call's "X-Trace" header).
 // If callback is provided & trace is sampled, cb will be called for entry event KVs
 func NewTraceFromID(spanName, mdStr string, cb func() KVMap) Trace {
-	return NewTraceFromIDForURL(spanName, mdStr, "", cb)
+	return NewTraceFromIDForURL(spanName, mdStr, "", Overrides{}, cb)
+}
+
+func NewTraceWithOverrides(spanName string, overrides Overrides, cb func() KVMap) Trace {
+	return NewTraceFromIDForURL(spanName, "", "", overrides, cb)
 }
 
 // NewTraceFromIDForURL creates a new Trace for the provided URL to report to AppOptics,
 // provided an incoming trace ID (e.g. from a incoming RPC or service call's "X-Trace" header).
 // If callback is provided & trace is sampled, cb will be called for entry event KVs
-func NewTraceFromIDForURL(spanName, mdStr string, url string, cb func() KVMap) Trace {
+func NewTraceFromIDForURL(spanName, mdStr string, url string, overrides Overrides, cb func() KVMap) Trace {
```

Comment:
From what I can see this is the only AO API with a changed signature, would any existing customer application using this function break?